src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -0,0 +1,342 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+#nullable enable
+
+using System;
+using System.Collections.Immutable;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+{
+    public class CodeGenFunctionPointersTests : CSharpTestBase
```

Comment:
>CodeGenFunctionPointersTests [](start = 17, length = 28)

It might be useful to have targeted tests for metadata import from IL to ensure that things end up in the right places.